build.gradle

```diff
@@ -179,7 +179,7 @@ dependencies {
     compile "org.springframework.cloud:spring-cloud-starter-sleuth:2.2.5.RELEASE"
     compile 'org.broadinstitute.dsde.workbench:sam-client_2.12:0.1-343dfff-SNAP'
     compile 'org.antlr:ST4:4.3'                          // String templating
-    compile group: 'io.kubernetes', name: 'client-java', version: '8.0.2'
+    compile group: 'io.kubernetes', name: 'client-java', version: '10.0.0'
```

Comment:
Is this needed in the main project?

Reply:
No, this ticket is for errors in the datarepo-clienttests project, not the main one. I just bumped this one too per an earlier [comment](https://github.com/DataBiosphere/jade-data-repo/pull/725#discussion_r532974698) on this PR.

Reply:
Ah got it. Just checking :)